### PR TITLE
Reverting Enable/Disable App Function Changes - Force `-net` App Function To Only Run `Agent` Functions

### PR DESCRIPTION
**Pull request by nharper285**
## Summary of the Pull Request

_What is this about?_
Having to update the disable logic to disable all functions in the `-net` app function except the agent functions. 

## PR Checklist
* [ ] Applies to work item: #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

### CODE REVIEW

**Review by AdamL-Microsoft — APPROVED** [review]
These look correct for the minimal needed to enable agent functionality on two dotnet functionapps.

File: src/ApiService/ApiService/Functions/ReproVmss.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ public class ReproVmss {
         _context = context;
     }
 
-    [Function("repro_vms")]
+    [Function("ReproVmss")]
```
Comment (Porges) [inline src/ApiService/ApiService/Functions/ReproVmss.cs:18]:
```suggestion
    [Function("ReproVms")]
```

File: src/ApiService/ApiService/Functions/Proxy.cs [hunk 2]
```diff
@@ -16,7 +16,7 @@ public class Proxy {
         _context = context;
     }
 
-    [Function("proxy")]
+    [Function("Proxy")]
     public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "GET", "PATCH", "POST", "DELETE")] HttpRequestData req) {
```
Comment (mgreisen) [inline src/ApiService/ApiService/Functions/Proxy.cs:20]: We'll need a route added here

File: src/ApiService/ApiService/Functions/ReproVmss.cs [hunk 3]
```diff
@@ -15,7 +15,7 @@ public class ReproVmss {
...
     public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "GET", "POST", "DELETE")] HttpRequestData req) {
```
Comment (mgreisen) [inline src/ApiService/ApiService/Functions/ReproVmss.cs:19]: And a route here as well

File: src/ApiService/ApiService/Functions/Proxy.cs (hunk 2 above)
Comment (nharper285) [reply src/ApiService/ApiService/Functions/Proxy.cs:20]: Why?

File: src/ApiService/ApiService/Functions/ReproVmss.cs (hunk 3 above)
Comment (Porges) [inline src/ApiService/ApiService/Functions/ReproVmss.cs:19]:
```suggestion
    public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "GET", "POST", "DELETE", Route = "repro_vms")] HttpRequestData req) {
```

Comment (nharper285) [reply src/ApiService/ApiService/Functions/ReproVmss.cs:19]: Wait why? This isn't necessary for the other functions.

File: src/ApiService/ApiService/Functions/Proxy.cs (hunk 2 above)
Comment (mgreisen) [reply src/ApiService/ApiService/Functions/Proxy.cs:20]: Or maybe not if George is right the matching is case-insensitive.

File: src/ApiService/ApiService/Functions/NodeAddSshKey.cs [hunk 4]
```diff
@@ -46,8 +46,8 @@ public class NodeAddSshKey {
 
     }
 
-    [Function("node_add_ssh_key")]
-    public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "node/add_ssh_key")] HttpRequestData req) {
+    [Function("NodeAddSshKey")]
+    public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "node_add_ssh_key")] HttpRequestData req) {
```
Comment (Porges) [inline src/ApiService/ApiService/Functions/NodeAddSshKey.cs:50]: Keep this one, we already had it explicitly specified

```suggestion
    public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "node/add_ssh_key")] HttpRequestData req) {
```

File: src/ApiService/ApiService/Functions/WebhookLogs.cs [hunk 5]
```diff
@@ -15,9 +15,9 @@ public class WebhookLogs {
         _context = context;
     }
 
-    [Function("Webhook_logs")]
+    [Function("WebhookLogs")]
     public Async.Task<HttpResponseData> Run(
-        [HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "webhooks/logs")] HttpRequestData req) {
+        [HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "webhooks_logs")] HttpRequestData req) {
```
Comment (Porges) [inline src/ApiService/ApiService/Functions/WebhookLogs.cs:20]:
```suggestion
        [HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "webhooks/logs")] HttpRequestData req) {
```

File: src/ApiService/ApiService/Functions/WebhookPing.cs [hunk 6]
```diff
@@ -15,9 +15,9 @@ public class WebhookPing {
         _context = context;
     }
 
-    [Function("Webhook_ping")]
+    [Function("WebhookPing")]
     public Async.Task<HttpResponseData> Run(
-        [HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "webhooks/ping")] HttpRequestData req) {
+        [HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "webhooks_ping")] HttpRequestData req) {
```
Comment (Porges) [inline src/ApiService/ApiService/Functions/WebhookPing.cs:20]:
```suggestion
        [HttpTrigger(AuthorizationLevel.Anonymous, "POST", Route = "webhooks/ping")] HttpRequestData req) {
```

File: src/ApiService/ApiService/Functions/ReproVmss.cs [hunk 7]
```diff
@@ -15,8 +15,8 @@ public class ReproVmss {
         _context = context;
     }
 
-    [Function("repro_vms")]
-    public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "GET", "POST", "DELETE")] HttpRequestData req) {
+    [Function("ReproVms")]
```
Comment (chkeita) [inline src/ApiService/ApiService/Functions/ReproVmss.cs:18]: this file can be renames to ReproVms.cs

**Review by chkeita — APPROVED** [review]

**Review by mgreisen — APPROVED** [review]
:shipit:

**Pull request merged**